benchmark/fixtures/simple-http-server.js

```diff
@@ -10,8 +10,9 @@ const storedUnicode = Object.create(null);
 const useDomains = process.env.NODE_USE_DOMAINS;
 
 // Set up one global domain.
+let domain;
```

Comment:
This one would probably be better as `const domain = require('domain');` I think? (Unless requiring the `domain` module has an undesirable side-effect? I don't think so, though.)

Reply:
Oh, wait, it's a fixture, so requiring an extra module could affect the performance. Meh, let's leave it as is, I guess. I'll delete this comment and my previous one.

Reply:
Er, or rather, I'll hide them rather than delete them.

Reply:
It has the side effect of always enabling async_hooks, which does affects performance non-trivially.